core/domain/value_generators_domain.py

```diff
@@ -26,23 +26,23 @@
 from core import feconf
 from core import utils
 
+from typing import Any, Dict, Tuple, Type
+
 
 class BaseValueGenerator:
     """Base value generator class.
-
```

Comment:
Why is this removed?

Reply:
@vojtechjelinek  I got a lot of style errors while pushing the first time. To fix them, I ended up modifying the docstrings. I will re-insert these newlines.